src/test/java/com/team3/central/services/CategoriesServiceTest.java

```diff
@@ -95,12 +101,10 @@ void notExistsByName() {
     assertThat(result).isFalse();
   }
 
-
-
   @Test
   public void shouldReturnCategoriesWithValidIds() {
     List<Integer> categoriesIds = List.of(1, 2);
-    Set<Category> expectedCategories = Set.copyOf(categories);
+    Set<Category> expectedCategories = Set.copyOf(getCategories());
```

Comment:
You can just do Set<Category> expected Categories = getCategories() no need for copying